sys/net/routing/rpl/rpl_storing/rpl_storing.c

```diff
@@ -149,7 +149,7 @@ static rpl_dao_t *get_rpl_dao_buf(void)
 
 static rpl_dao_ack_t *get_rpl_dao_ack_buf(void)
 {
-    return ((rpl_dao_ack_t *) & (buffer[(LL_HDR_LEN + IPV6_HDR_LEN + ICMPV6_HDR_LEN)]));
```

Comment:
Is this a bug?

Is it supposed to be `sixlowpan_buffer[(LL_HDR_LEN + IPV6_HDR_LEN + ICMPV6_HDR_LEN)]` or `rpl_buffer[IPV6_HDR_LEN + ICMPV6_HDR_LEN]`?